bigtable/test/BigtableTestTrait.php

```diff
@@ -96,7 +96,7 @@ public static function deleteBigtableInstance()
         self::$instanceAdminClient->deleteInstance($instanceName);
     }
 
-    private static function runSnippet($sampleName, $params = [])
+    private static function runFunctionSnippet($sampleName, $params = [])
```

Comment:
There is a function `runFunctionSnippet` built into `TestTrait` which should be handling this.
```suggestion
    private static function runSnippet($sampleName, $params = [])
```